listenbrainz/webserver/templates/index/playlists-management.html

```diff
@@ -0,0 +1,34 @@
+{%- extends 'base.html' -%}
+{%- block title -%}Playlists management - ListenBrainz{%- endblock -%}
+{%- block content -%}
+  <h2 class="page-title">Playlists managementg </h2>
+  {# Playlists management information #}
+  <h3>Synchronizing Playlists</h3>
+  <p>
+    There are many ways to sync your own playlists to ListenBrainz:
+  </p>
+  <h4>Music players</h4>
+  <ul>
+    <li><em><a href="https://www.foobar2000.org/">Foobar2000</a></em>, full-fledged audio player for Windows: <a href="https://github.com/regorxxx/Playlist-Manager-SMP"><code>Playlist-Manager-SMP</code></a></li>
+  </ul>
+
+  <h3>Synchronizing via third-party music services</h3>
+  <p>
+    <a href={{ url_for('profile.music_services_details') }}>Connect your ListenBrainz account</a> and import/export your playlists to some third-party services.
+  </p>
+  <ul>
+    <li><em><a href="https://open.spotify.com/">Spotify</a></em></li>
```

Comment:
I think here what would be more useful would be third-party services mentioned above that allow you to export and/or transform playlists.

Looking around quickly for OSS I found  https://github.com/watsonbox/exportify, as well as some non-open-source options like https://www.tunemymusic.com and https://freeyourmusic.com that can export to CSV

There are also tools that allow to transform other formats to XSPF such as https://github.com/kevlened/csvToXspf , and there must be good OSS alternatives out there to do the same for M3U playlists and other standard playlist formats

Reply:
Although I agree it would be great to at least link to services like those (specially to give people tools to make use of their spotify data for open source services), there is a caveat: current problem of ListenBrainz is not being able to export the JSPF playlists in an user-friendly way.

Reply:
I mean, even if I add more info about what to do with Spotify playlists, currently there is only one software example being  able to export ListenBrainz playlists to working files. And that's the weak part.

Reply:
I'll create a ticket for exporting playlists in an easier manner

Reply:
**Btw, note I already coded a JSPF/XSPF library in javascript, which may be reused by the team with minimal changes.**

XSPF parser to JSPF
https://github.com/regorxxx/Playlist-Manager-SMP/blob/main/helpers-external/xspf-to-jspf-parser/xspf_parser.js
JSPF parser to XSPF
https://github.com/regorxxx/Playlist-Manager-SMP/blob/main/helpers/helpers_xxx_playlists_files_xspf.js

Only change required on your side would be merging both files, and scrapping the "include" part.

Also when exporting to XSPF or JSPF, it would be great to add as location the youtube link found on the ListenBrainz player, as commented below. Not sure if at least the current youtube content resolution could be used when exporting, since it's already coded. It could mean hundreds of queries at once though.

Reply:
Love the *SPF libraries, that could indeed come in very handy :)

If I'm honest, I'm not at all impressed by the youtube search results when used in the context of the ListenBrainz player. Since the youtube API tries to return a result, any result rather than none, we often get results that are quite trash. I wouldn't rely on it for a good playlist…
However when we do have a better form of content resolution (we're keen on this for obvious reasons) we'll be sure to include the track locations in the playlists.